lib/dialogs/unsynchronized/style.scss

```diff
@@ -31,6 +31,7 @@
   .change-list {
     border-bottom: 1px solid $studio-gray-5;
     border-top: 1px solid $studio-gray-5;
+    overflow-y: auto;
```

Comment:
This is the only change from this PR, the rest are coming from the original [PR](https://github.com/Automattic/simplenote-electron/pull/2815).